optimum/intel/openvino/modeling.py

```diff
@@ -371,23 +373,27 @@ class OVModelForFeatureExtraction(OVModel):
 
     def __init__(self, model=None, config=None, **kwargs):
         super().__init__(model, config, **kwargs)
+        
+        from sentence_transformers import SentenceTransformer
```

Comment:
it is also good to add checks that sentence transformers available here and check it before imports https://github.com/huggingface/optimum-intel/blob/main/optimum/intel/utils/import_utils.py